doc/15-troubleshooting.md

```diff
@@ -317,6 +317,52 @@ not `assign where match("MySQL*", host.name) && host.vars.production`.
 For all test hosts `host.vars.production` will be false and terminate
 the `&&` rather than also evaluating `match("MySQL*", host.name)`.
 
+### Try reducing concurrency (threads)
+
+!!! note
+
+	This section applies rather to systems with more than eight CPU cores.
+	In case of less consider upgrading your hardware instead.
+
+Yes, reducing and not increasing. By default, Icinga 2 already starts as many
+threads as there are CPU cores according to the OS (unaware of SMT aka
+Hyper-Threading). So there's no point in increasing as Icinga would not gain
+additional CPU time.
+
+But more threads also require more synchronization between them. This may
+outweigh the CPU time gained by starting X threads instead of just one. So
```

Comment:
"X threads instead of just one" sounds a bit like the alternative would be no multithreading at all?

What do you think of some working like this? "With a sufficiently large number of threads, adding more threads may no longer provide a speedup, or worse, it may even slow things down due to growing synchronization overhead."

Reply:
You're right, that fragment indeed only confuses the reader at no benefit.